gradle/verification-metadata.xml

```diff
@@ -5614,6 +5614,17 @@
             <sha256 value="8314ff61776036dacab235bf0a225d4ef901e1555a86f91977e162780d3bbe54" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="tech.pegasys" name="jc-kzg-4844" version="0.3.0">
```

Comment:
do we need a process with this verification metadata generation to remove the old metadata eg
`       <component group="tech.pegasys" name="jc-kzg-4844" version="0.1.0">`